libclc/amdgcn-amdhsa/libspirv/math/nextafter.cl

```diff
@@ -0,0 +1,11 @@
+ #include <spirv/spirv.h>
+ #include <clcmacro.h>
+ 
+ double __ocml_nextafter_f64(double,double);
```

Comment:
clang-format here and the next line

Reply:
Sorry，I delete my repository by mistake, and I forked again. But I don't know how to update this request now, should I close this one ,and create a new pull request?